supporting_content/appendix/quantity_driven_pricing_strategies.md

```diff
@@ -38,49 +38,49 @@ For better comprehension, please refer to the sample price-tiers configuration a
 
 #### UC scenarios for different (sub)types of pricing mechanisms
 
-| Scenario | Description                                                                                                 |
-|:---------|:------------------------------------------------------------------------------------------------------------|
-| S-1      | Tier-based pricing: 12GB of usage falls into two different tiers, resulting in two separate charge records. |
-| S-2      | Volume-based pricing: 12GB of usage falls into the highest tier, resulting in a single charge record.       |
+| Scenario | Description                                                                                              |
+| :------- | :------------------------------------------------------------------------------------------------------- |
+| S-1      | Tier-based pricing: 12GB of usage falls into two different tiers, resulting in two separate charge rows. |
```

Comment:
This is the only change in this file. The formatting changes were auto-applied on save.